### ISSUE TITLE
[SPOILER ALERT!!!] ATTENTION PLEASE. IT'S IMPORTANT TO YOU TO KNOW.

### ISSUE THREAD

**Issue by WayneSkywalker** [narze/thwordle#34]
Don't save answers into a file. Please save into the database or somewhere else.
I'm sorry if it's by designed.

**Comment by PaperMonster** [comment]
I agree. Browser caching problem is troubling many people.

**Comment by narze** [comment]
It's by design, Wordle did this as well.

![image](https://user-images.githubusercontent.com/248741/156705488-d91d89af-f1b6-4bf4-b903-4de8ff51afd8.png)

Using database and API will certainly exceed the free quota.

I've changed the wordset only once to add more words, and I won't do it ever again.
